### PR TITLE
Improve the markup of powers in docs

### PR DESCRIPTION
**Pull request by serhiy-storchaka**

### CODE REVIEW

File: Doc/library/ipaddress.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ IP addresses, networks and interfaces:
 
    Return an :class:`IPv4Address` or :class:`IPv6Address` object depending on
    the IP address passed as argument.  Either IPv4 or IPv6 addresses may be
-   supplied; integers less than 2**32 will be considered to be IPv4 by default.
+   supplied; integers less than ``2**32`` will be considered to be IPv4 by default.
```
Comment (serhiy-storchaka) [inline Doc/library/ipaddress.rst:44]: See how the GitHub syntax highlighter is confused with the old text.

File: Doc/library/functions.rst [hunk 2]
```diff
@@ -1345,8 +1345,8 @@ are always available.  They are listed here in alphabetical order.
    coercion rules for binary arithmetic operators apply.  For :class:`int`
    operands, the result has the same type as the operands (after coercion)
    unless the second argument is negative; in that case, all arguments are
-   converted to float and a float result is delivered.  For example, ``10**2``
-   returns ``100``, but ``10**-2`` returns ``0.01``.
+   converted to float and a float result is delivered.  For example, ``pow(10, 2)``
+   returns ``100``, but ``pow(10, -2)`` returns ``0.01``.
```
Comment (ambv) [inline Doc/library/functions.rst:1349]: Cool, I always found it weird not to use the function in question in those examples.

**Comment by ambv** [comment]
I'll deal with the conflicts.

**Comment by serhiy-storchaka** [comment]
Thank you for backports @ambv.

**Pull request merged**